### PR TITLE
fix(modeling): ensure group min dimensions

### LINKED ISSUE #1705: Resizing Group Using Space Tool Breaks Application
__Describe the Bug__

__Steps to Reproduce__

1. Create new bpmn diagram on https://demo.bpmn.io/
2. Add group
3. Activate space tool and use on group
4. Move group with space tool up
5. Space tool stuck and can be dropped only after 2 clicks
6. After drop all elements can't move
[](url)

__Expected Behavior__

Space tool don't stuck and elements can move

__Environment__

 - Browser: Chrome 101, Yandex browser 21.2
 - OS: Mac and Win 10
 - Library version: v9.3.0
[](url)

### PR DESCRIPTION
**Pull request by philippfromme**
brave_3neQFwZF9M

Closes #1705

### CODE REVIEW

**Comment by nikku** [comment]
I believe this has been a source of bug reports / errors. Great find!

File: test/spec/features/modeling/behavior/SpaceToolBehaviorSpec.js [hunk 1]
```diff
@@ -149,4 +151,36 @@ describe('features/modeling - space tool behavior', function() {
...
+  describe('group', function() {
+
+    describe('minimum dimensions', function() {
+
+      var diagramXML = require('./SpaceToolBehaviorSpec.group.bpmn');
+
+      beforeEach(bootstrapModeler(diagramXML, { modules: testModules }));
+
+
+      it('should ensure subprocess minimum dimensions', inject(
```
Comment (nikku) [inline test/spec/features/modeling/behavior/SpaceToolBehaviorSpec.js:164]:
```suggestion
      it('should ensure group minimum dimensions', inject(

```

**Comment by pinussilvestrus** [comment]
Now I even know where lots of [postit-js](https://github.com/pinussilvestrus/postit-js) errors come from (I simply copied the groups implementation) 🐵

**Comment by nikku** [comment]
HARHAHRA :cake:

File: test/spec/features/modeling/behavior/SpaceToolBehaviorSpec.js (hunk 1 above)
Comment (nikku) [reply test/spec/features/modeling/behavior/SpaceToolBehaviorSpec.js]: Corrected via https://github.com/bpmn-io/bpmn-js/pull/1713/commits/1c54e97fe34b66af0a69bbd4bdc2a4179b074364.

**Review by nikku — APPROVED** [review]
Just works! :muscle:

**Comment by nikku** [comment]
For whatever reason CI is lame again today. Let's merge it, because it works on Windows.

**Pull request merged**